components/admin_console/billing/billing_history.tsx

```diff
@@ -33,6 +34,7 @@ const noBillingHistorySection = (
             rel='noopener noreferrer'
             href={CloudLinks.BILLING_DOCS}
             className='BillingHistory__noHistory-link'
+            onClick={() => trackEvent('cloud_admin', 'click_billing_history', {screen: 'billing'})}
```

Comment:
I've been adding the event category to a list of constants (see [here](https://github.com/mattermost/mattermost-webapp/pull/6763/files#diff-f44101ff0bb1f1a11dbbb09428a2cf828fe2ae4b29dac499565b74946bf7d1f9R410))
I'm 0/5 on having to do this everywhere, but we should probably choose one way to do it (either yours or mine) and stick with it.

Reply:
We don't do it anywhere else in the system, you can do a quick search on `trackEvent` and it'll show you.
Not saying we shouldn't use constants, I definitely like the idea of doing so, but for consistencies sake we should probably just not do it until we do it everywhere. 2/5

Reply:
Fair. Mine are already in so will have to go back to adjust at some point